src/pretix/plugins/sendmail/tasks.py

```diff
@@ -76,7 +79,7 @@ def send_mails_to_orders(event: Event, user: int, subject: dict, message: dict,
                         list_id__in=checkin_lists or []
                     )
                 ),
-            ).prefetch_related('addons'):
+            ).prefetch_related('addons', 'subevent'):
```

Comment:
Probably not worth the optimization, but should we prefetch `subevent` only if at least one of the new subevent-parameters is provided?

Reply:
I think that is in fact not worth the complexity